tools/release.sh

```diff
@@ -151,6 +155,33 @@ sign() {
     fi
   done
 
+  while true; do
+    printf "Upload files from %s to R2 staging bucket? [y/n]" "$webhost"
```

Comment:
I just realized, this step should actually copy to the production bucket. `sign` happens after `promote`.